File: PrimeNumber.java
import java.util.Scanner;

public class PrimeNumber {
    public static void main(String[] args) 
    {
        Scanner scanner = new Scanner(System.in);
        int n;

        System.out.print("Enter the range of Array: ");
        n = scanner.nextInt();

        int[] a = new int[n];
        System.out.println("Enter the elements of Array: ");
        for (int i = 0; i < n; i++) 
        {
            a[i] = scanner.nextInt();
        }

        System.out.print("Prime numbers in the array are: ");
        for (int i = 0; i < n; i++) 
        {
            int item = a[i];
            if (prime(item)) 
            {
                System.out.print(item+" ");
            }
        }
        System.out.println();
    }

    public static boolean prime(int num) 
    {
        if (num <= 1) 
        {
            return false;
        }
        for (int i = 2; i <= Math.sqrt(num); i++) 
        {
            if (num % i == 0) 
            {
                return false;
            }
        }
        return true;
    }
}

File: PrimeNumber.py
import math

def is_prime(num):
    if num <= 1:
        return False
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            return False
    return True

n = int(input("Enter the range of Array: "))
a = []

print("Enter the elements of Array: ")
for _ in range(n):
    a.append(int(input()))

print("Prime numbers in the array are: ", end="")
for item in a:
    if prime(item):
        print(item, end=" ")
print()
